templates/partials/bbot-to-deriv-banner.mustache

```diff
@@ -0,0 +1,13 @@
+<div class="dbot-banner eu-hide invisible">
```

Comment:
Actually there is a MovingBanner component which is react based and you can use it

Reply:
thank you, i've seen it, but decided to copy the "try DBot" banner